_pages/dev/i18n.md

```diff
@@ -244,99 +248,103 @@ You can also pass parameters into the translation pipe, for example:
+
+For more information about different pluralization rules, see [Localization and Plurals](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals) on the Mozilla developer site.
 
-More about the special `count` parameter in the official `i18next` docs: https://www.i18next.com/translation-function/plurals
+You can also use this small tool to help you figure out the correct plural suffixes for different languages: [https://jsfiddle.net/jamuhl/3sL01fn0/#tabs=result](https://jsfiddle.net/jamuhl/3sL01fn0/#tabs=result)
 
 
-#### *context* parameter
+### Special Parameter: context
 
-You can pass the special parameter `context` to differ translations for the same key depending on the string value of `context`. It's useful to translate enum values from backend. For example:
+The `context` parameter is a special parameter of the translation pipe that is used for conditional translations.
+
+You can pass the special `context` parameter to differ translations for the same key depending on the string value of `context`. It is useful for translating enum values from the back end. The following is an example:
 
 {% raw %}
+
 ```html
 <p>{{ 'order.status' | cxTranslate : { context: order.status } }}</p>
 ```
+
 {% endraw %}
 
-Translation resources:
+The following are the relevant translation resources:
 
-```js
+```ts
 {
-    "order": {
-        "status_processing": "In process...",
-        "status_completed": "Completed",
-        "status_cancelled": "Cancelled",
+    order: {
+        status_processing: 'In process...',
+        status_completed: 'Completed',
+        status_cancelled: 'Cancelled',
     }
 }
 ```
 
-More about the special `context` parameter in the official i18next docs: https://www.i18next.com/translation-function/plurals
-
-### Other special parameters (unsupported)
+For more information about the special `context` parameter, see [Context](https://www.i18next.com/translation-function/context) in the official i18next documentation.
 
-There are few special parameters which interfere with the `i18next` translation mechanism, but Spartacus doesn't support them and we recommend to avoid using them:
+### Unsupported Special Parameters
 
-`defaultValue`, `replace`, `lng`, `lngs`, `fallbackLng`, `ns`, `keySeparator`, `nsSeparator`, `returnObjects`, `joinArrays`, `postProcess`, `interpolation`, `skipInterpolation`
+The following special parameters interfere with the `i18next` translation mechanism, and Spartacus does not support them: `defaultValue`, `replace`, `lng`, `lngs`, `fallbackLng`, `ns`, `keySeparator`, `nsSeparator`, `returnObjects`, `joinArrays`, `postProcess`, `interpolation`, `skipInterpolation`
 
-That being said, here are official `i18next` docs for those *unsupported* parameters: https://www.i18next.com/translation-function/essentials#overview-options
```

Comment:
I made the list of special parameters, based on the i18next official docs from at the day of writing. Meanwhile i18nect could have implemented more special parameters. The list can grow in the future (we don't control their API, but we proxy it with only a disclaimer that we don't take responsibility for all of their API).

I would vote for leaving the link to the official docs here as the future-proof source of truth for our customers. I don't mind however we rephrase it ;)